### PR TITLE
Add unit tests for the entity validators

### PR DESCRIPTION
**Pull request by Elhebert**
### What does it do?

It add unit test for the different validators we have for the entities.

### CODE REVIEW

File: packages/core/strapi/lib/services/entity-validator/__tests__/biginteger-validators.test.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+const strapiUtils = require('@strapi/utils');
+const { YupValidationError } = require('@strapi/utils/lib/errors');
+const entityValidator = require('../validators');
```
Comment (alexandrebodin) [inline packages/core/strapi/lib/services/entity-validator/__tests__/biginteger-validators.test.js:5]: shouldn't this be called validators ?

**Review by alexandrebodin — CHANGES_REQUESTED** [review]
That's looking very cool. 

Do we already have a test suite for the actually entity validator too ?

**Comment by Elhebert** [comment]
I had a look, but haven't seen anything. It might be tested through some of our e2e tho. But nothing specific.

I'll add some UT in there and will try to include as many cases as I can.

**Review by alexandrebodin — APPROVED** [review]
LGTM. Great improvements here

**Pull request merged**